design/README.md

```diff
@@ -44,37 +40,44 @@ The expanded version should be use to generate the SVG file.
 
```

Comment:
"SVG should not include `<title>` or `<defs>` tags." 
These two are not within the control of a designer (unless you open SVG with some text editor). I (1) tried really hard to have AI generate an SVG without title or def and had no success and (2) found some existing icons with the titles attribute already. I'd say remove this instruction

Reply:
> Be sure to include a 24x24px rectangle with no stroke and no fill to serve as the bounding box.

I personally did not follow this last time when I was making changes in this repo. (1) we had to manually remove the bounding box when we import them into Figma; (2) the same functionality can be achieved by checking "use artboards" option when the saving/file naming dialog box pop up (so not part of the export options window, which appears in the next screen)

Reply:
The 24px bounding box is necessary to make the entire 24x24 square clickable...if we leave it off, icons like the three-dot menu are almost impossible to click because you have to hit directly on the path.

Reply:
(1) Our event listener is on the container surrounding the `<svg>` tag, not on the `<path>` or `<svg>`
(2) I randomly checked three icons in the exported [svg folder](https://github.com/pxblue/icons/tree/dev/svg) (health, barley, flask), neither of which has a bounding box thing in the source code. I think as long as the artboard/view box ended up doing 24*24, it should be OK

Reply:
I agree, most of the time you can wrap the icon in something else and put the click handler there (but you don't have to).

I checked all of the icons: 109/139 have the bounding box. If we're going to change it, we should make them all the same. We can do that in a separate issue.